### PR TITLE
[REVIEW] - Update docs for 0.15 release

### PR DESCRIPTION
**Pull request by ajschmidt8**
This PR updates the docs for the 0.15 release.

Also, commit https://github.com/rapidsai/docs/pull/114/commits/c0a77ec50ca5b2a6a5b81abba181ed60b395f1aa includes a fix for #105. This was done by manually checking out the [v0.2.7 version](https://github.com/pmarsceill/just-the-docs/releases/tag/v0.2.7) of [just-the-docs](https://github.com/pmarsceill/just-the-docs) and compiling the SCSS to CSS and then replacing our current search styles with the new manually compiled styles.

### CODE REVIEW

**Comment by ajschmidt8** [comment]
Closes #105

File: maintainers/maintainers.md [hunk 1]
```diff
@@ -27,20 +27,18 @@ Operations
...
 Release | Wed, Oct 14th | Wed, Oct 14th | 1 day
+
+## _PROPOSED_ - Release v0.17 Schedule
+
+Phase | Start | End | Duration
+-- | -- | -- | --
+Development | Thu, Oct 1st | Mon, Nov 23rd | 38 days
+Burn Down | Tue, Nov 24th | Wed, Dec 2nd | 5 days
+Code Freeze/Testing | Thu, Dec 3rd | Tue, Dec 8th | 4 days
+Release | Wed, Dec 9th | Wed, Dec 9th | 1 day
```
Comment (mike-wendt) [inline maintainers/maintainers.md:44]: I'd drop this for now as we haven't set this in stone. Also v0.16 is still in flux and won't be set until the meeting tomorrow. So let's add back _PROPOSED_ to v0.16

**Comment by mike-wendt** [comment]
@ajschmidt8 can you update the docker file [description](https://github.com/rapidsai/docs/blob/release-0.15/demos/rapids-demo.md) as well? It still refers to python 3.6 in some places

File: maintainers/maintainers.md (hunk 1 above)
Comment (ajschmidt8) [reply maintainers/maintainers.md]: done

**Comment by mike-wendt** [comment]
Will merge when release is done

**Pull request merged**